broker/src/main/java/io/camunda/zeebe/broker/partitioning/PartitionFactory.java

```diff
@@ -56,10 +57,10 @@
 import java.util.function.Consumer;
 import java.util.stream.Collectors;
 
-final class PartitionFactory {
-  // preparation for future steps
-  // will be executed in the order they are defined in this list
-  private static final List<PartitionStartupStep> STARTUP_STEPS = List.of();
+// TODO make package private again
+public final class PartitionFactory {
+
+  public static final boolean FEATURE_TOGGLE_USE_NEW_CODE = false;
```

Comment:
Let's add a short comment here so that others reading the code doesn't get confused.